frontend/packages/topology/src/components/quick-search/QuickSearchList.tsx

```diff
@@ -0,0 +1,117 @@
+import * as React from 'react';
+import { useTranslation } from 'react-i18next';
+import cx from 'classnames';
+import { history, removeQueryArgument } from '@console/internal/components/utils';
+import { CatalogItem } from '@console/plugin-sdk';
+import {
+  DataList,
+  DataListItem,
+  DataListItemRow,
+  DataListItemCells,
+  DataListCell,
+  Split,
+  SplitItem,
+  Label,
+  TextContent,
+  Text,
+  TextVariants,
+  Button,
+} from '@patternfly/react-core';
+import './QuickSearchList.scss';
+
+interface QuickSearchListProps {
+  listItems: CatalogItem[];
+  totalItems: number;
+  selectedItemId: string;
+  searchTerm: string;
+  namespace: string;
+  onSelectListItem: (itemId: string) => void;
+}
+
+const QuickSearchList: React.FC<QuickSearchListProps> = ({
+  listItems,
+  totalItems,
+  selectedItemId,
+  searchTerm,
+  namespace,
+  onSelectListItem,
+}) => {
+  const { t } = useTranslation();
+  const ref = React.useRef(null);
+
+  const openForm = (e: React.SyntheticEvent) => {
+    e.persist();
+    e.stopPropagation();
```

Comment:
Not sure you need `stopPropagation` 🤔 

Who else is catching the double-click?

Reply:
I had to add this to stop the event to propagate to its children since if it does the target item (which in this case I always want it to be the `li` element since from there I get the UID of the item) changes based on where you are clicking inside the list item block.

Reply:
You're going to have to help me understand this... I am not following 🙂 

`stopPropagation` prevents event bubbling from continuing. Events bubble from the touched item (text, icon, etc) up indefinitely until they hit the top of the document. If we say had this appear as a node in Topology, one could want to stop propagation on double clicks to avoid the Topology Graph from zooming... but I don't know what is happening here that we want to stop 🤔

Reply:
Yeah you are absolutely right. But initially if I clicked on for example the label inside the list item, current of the ref was not `li` but that `label` and thus navigation was failing but when I added `stopPropagation` current was always `li` and it somehow worked. But now again when I removed it, everything seems to work fine :see_no_evil: If you see any issues with double click now please let me know.